Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -36,6 +36,8 @@
 #include "DolphinQt/QtUtils/ModalMessageBox.h"
 #include "DolphinQt/Settings.h"
 
+typedef MemoryViewWidget::Type InputID;
```

Comment:
I think it would be less confusing to just update all instances of InputID, but if you want to keep this route, please use `using InputID = MemoryViewWidget::Type`.

Reply:
I just noticed it sometimes references MemoryViewWidget::Type already.  Would `using Type = MemoryViewWidget::Type` be okay or should I just stick to the long version?  I have to drop InputID either way.

Reply:
Yeah `using Type = MemoryViewWidget::Type` should be fine here.